src/utils/embeds.ts

```diff
@@ -0,0 +1,30 @@
+import { Client, MessageEmbed } from "discord.js";
+/**
+ * @description Creates a default embed for bot messages
+ * @author John Schiltz
+ * @export
+ * @param client - Discord Client
+ * @param title - Title of embed
+ * @param description - Description of embed
+ * @return {*} - MessageEmbed
+ */
+export function create_default_embed(
+  client: Client,
+  title: string,
+  description: string
+) {
+  const color = "#0099ff";
+  const thumbnail =
+    "https://playantares.com/resources/CSSC-bot/cssc-server-icon.png";
```

Comment:
domain should be changed to `antaresnetwork.com`

Reply:
for some reason the change from staging didn't come through, so I fixed all 13 occureneces